man/journalctl.xml

```diff
@@ -749,6 +749,18 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--namespace=<replaceable>NAMESPACE</replaceable></option></term>
```

Comment:
Can I do something like: `+webserver,db,msgbus` and have those three namespaces added to the default shown?

Reply:
nope, currently you specify one namespace and then optionally get the default one added in, that's all. Extending this eventually to a list of namespaces is pretty easy though, but let's do that later.